helix-rest/src/main/java/org/apache/helix/rest/server/ServerContext.java

```diff
@@ -278,6 +282,17 @@ public HelixDataAccessor getDataAccessor(String clusterName) {
     return _byteArrayZkBaseDataAccessor;
   }
 
+  public ZkBucketDataAccessor getZkBucketDataAccessor() {
```

Comment:
Since this is not used right now, shall we put it in the next PR?

Reply:
Yea make sense.